omnipaxos_core/src/sequence_paxos/follower.rs

```diff
@@ -100,6 +101,12 @@ where
             self.internal_storage.set_accepted_round(accsync.n);
             self.internal_storage.set_decided_idx(accsync.decided_idx);
             self.state = (Role::Follower, Phase::Accept);
+            // Begin accept sequence
+            if self.accept_sequence != 0 {
+                panic!("AcceptSync must be first message in accept sequence!");
+            }
```

Comment:
Since we have passed the check that we are in the prepare phase (where we set the seq number to 0), this can be removed.

```suggestion
```